go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/elastic/elastic-agent
 
-go 1.22.9
+go 1.22.10
```

Comment:
I believe we shouldn't bump golang version in this PR, right? 🙂

Reply:
yeah, not needed. I've removed it.